src/owasp-dependency-check-suppressions.xml

```diff
@@ -181,6 +181,16 @@
         <sha1>fa9a1ccda7d78edb51a3a33d3493566092786a30</sha1>
         <cve>CVE-2021-25263</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+        file name: clickhouse-jdbc-0.4.6-all.jar
```

Comment:
```
Error:  One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
Error:  
Error:  canal.client-1.1.5.jar/META-INF/maven/com.google.guava/guava/pom.xml: CVE-2023-2976(7.1)
Error:  clickhouse-jdbc-0.4.6-all.jar/META-INF/maven/com.google.guava/guava/pom.xml: CVE-2023-2976(7.1)
```

It seems this patch doesn't suppress the false positive.

Reply:
I'll trigger one. But generally you can test in your personal fork and that should be the same.